### DISCUSSION TITLE
Contributing to keycloakify

### DISCUSSION THREAD

**Discussion by giorgoslytos** [keycloakify/keycloakify#501] [Q&A] [answered]
Hi @garronej !

Thank you for your work on Keycloakify!

It has been very useful to us and I would like to contribute to the project!
However I still face issues on setting the development environment.
I have followed the contributing documentation, but failed to make it work.

Shouldn't the watcher be able to see changes in src/login/pages? It is listening only on src/lib and src/bin.
Also on the starter project how do I get pages from the keycloakify project I just edited?
I will have to build it firstly and then import it somehow to the keycloakify-starter project?
In the case of adding or editing pages the watch mode is useless as well as the link between keycloakify and the starter project?

Thank you again!

**Comment by garronej** [comment, marked as answer]
Hello @giorgoslytos,  
Thanks for your willingness to contribute!  
If you want I can take a few minutes to help you get started.

**Comment by garronej** [comment]
You can join the discord with this link: https://discord.gg/kYFZG7fQmn
